server/adaptors/integrations/__data__/repository/aws_rds/info/README.md

```diff
@@ -0,0 +1,25 @@
+# AWS RDS Integrations
+
+## What is AWS RDS?
+AWS RDS (Relational Database Service) is a managed service that makes it easy to set up, operate, and scale a relational database in the cloud.
+
+RDS helps you perform tasks such as:
+
+* Managing database instances
+* Scaling compute resources and storage capacity
+* Automating time-consuming administration tasks including hardware provisioning, database setup, patching, and backups
+
+RDS keeps your database up-to-date with the latest patches, and it also provides automatic backups and disaster recovery capabilities. You can make database instances available in multiple regions to enhance availability and reliability for your data.
+
+See additional details [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.html).
+
+## What is AWS RDS Integration?
+An integration is a collection of predefined assets which are combined in a meaningful manner.
+
+AWS RDS integration includes dashboards, visualizations, queries, and index mapping to help you manage and monitor your database services more effectively.
```

Comment:
I think these READMEs would benefit a lot from a brief discussion of how to set up the service for locally testing the integration. People trying to integrate with RDS know what RDS is, but don't necessarily know how they get from point A to point B.